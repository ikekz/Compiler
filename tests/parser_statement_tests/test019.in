var
	a : array of array of integer;
	b: array of integer;
begin
	a[8] := b;
end.