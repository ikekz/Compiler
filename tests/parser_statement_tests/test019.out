begin
    :=
        []
            a
            8
        b
end
